portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionUsingConfigFile.testcase

```diff
@@ -88,9 +88,24 @@ definition {
 	@description = "Verify that the Workspace label is present when a client extension is created through osgi"
 	@priority = 3
 	test OsgiClientExtensionWorkspaceLabelIsPresent {
+		task ("Given config file for client extension of type custom element created") {
+			ClientExtension.createGlobalVariableClientExtensionType(type = "customElement");
+
+			ClientExtension.checkConfigFileCreatedProperly();
+		}
+
+		task ("When Go to applications/custom apps/client extensions") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Custom Apps",
+				panel = "Applications",
+				portlet = "Client Extensions");
+		}
 
-		// TODO LPS-171621 OsgiClientExtensionWorkspaceLabelIsPresent pending implementation
+		task ("Then (date column) and label (workspace) are present") {
+			AssertTextPresent(value1 = "Modified Date");
 
+			AssertTextPresent(value1 = "Workspace");
```

Comment:
You need to use proper xpath locators, having only value1 is not enough for AssertTextPresent.

Reply:
Or could you use `FDSCustomView.assertColumnPresent(key_field = "Modified Date");` for asserts.

Reply:
I think the same macros here will work: https://github.com/liferay-frontend/liferay-portal/pull/3039#issuecomment-1426222861

Reply:
This tip is important!!
thanks John